src/tests/dfio/SConscript

```diff
@@ -0,0 +1,34 @@
+"""Build daos fio engine"""
+from __future__ import print_function
+import os
+import daos_build
+
+def scons():
+    """Execute build"""
+
+    Import('env', 'prereqs')
+
+    libs = ['daos_common', 'daos', 'dfs', 'gurt', 'uuid', 'dl']
+
+    denv = env.Clone()
+
+    if prereqs.has_source(env, 'fio'):
+        print("Building DAOS FIO Engine with source dir", env.subst("$FIO_SRC"))
+    else:
+        print("Skipping DAOS FIO Engine build")
+        return
+
+    denv.AppendUnique(CPPPATH=["$FIO_SRC"])
+    denv.Append(CCFLAGS="-U_FORTIFY_SOURCE -D_FORTIFY_SOURCE=0 -O0")
```

Comment:
-include config-host.h should probably be added.  It is the result of fio configure script and is passed to their builds